hack/run-functests.sh

```diff
@@ -1,15 +1,20 @@
 #!/bin/bash
 set -x
-which ginkgo
+# Install Go 1.20.4
+wget https://go.dev/dl/go1.20.4.linux-amd64.tar.gz
+rm -rf /usr/local/go
```

Comment:
If this is a temporary workaround please add a `TODO` comment. For a permanent solution, I suggest 1. check the existing go version and install newer version only when needed. 2. install new go in {ptp-operator repo}/bin and add this in front of the PATH.

Reply:
I will have a subsequent update to address your comment. Trying to enable CI pipeline again with this change.